internal/configure/hosts/hosts.go

```diff
@@ -107,14 +117,137 @@ func (hc *HostConfig) convertEnvs() error {
 			hc.envs = append(hc.envs, v)
 		}
 	}
+	return nil
+}
+
+// read the instances value from hc.config
+func (hc *HostConfig) convertInstances() error {
+	value := hc.config[KEY_INSTANCES]
+	v, ok := utils.All2Str(value)
```

Comment:
Maybe `if v, ok := utils.All2Str(value);!ok` is better. Keep your code style consistent.

Reply:
for example,
```go
func (hc *HostConfig) convertInstances() error {
	value := hc.config[KEY_INSTANCES]
	if instancesStr, ok := utils.All2Str(value);!ok {
		return errno.ERR_UNSUPPORT_CONFIGURE_VALUE_TYPE.
			F("hosts[%d].%s = %v", hc.sequence, KEY_INSTANCES, value)
	}
	if instances, ok := utils.Str2Int(instancesStr); !ok {
		return errno.ERR_CONFIGURE_VALUE_REQUIRES_INTEGER.
			F("hosts[%d].%s = %v", hc.sequence, KEY_INSTANCES, value)
	}
	if instances <= 0 {
		return errno.ERR_CONFIGURE_VALUE_REQUIRES_POSITIVE_INTEGER.
			F("hosts[%d].%s = %v", hc.sequence, KEY_INSTANCES, value)
	}
	hc.instances = instances
	return nil
}
```